templates/taxi/driver_detail.html

```diff
@@ -4,7 +4,12 @@
   <h1>
     Username: {{ driver.username }}
   </h1>
-
+  <a href="{% url 'taxi:driver-delete' pk=driver.id %}" class="btn btn-danger link-to-page">
+      Delete
+    </a>
```

Comment:
According to the checklist, you should add a `Cancel` button apart from the `Delete` one. The `Cancel` button should lead to the previous page the user was on.